_terms/cookie-notice.md

```diff
@@ -1,49 +1,49 @@
 ---
 layout: article
-title: SixSq Website Cookie Notice
-subtitle: This Cookie Notice describes how we SixSq Sarl (SixSq or we) use cookies and similar tracking technologies on our web site to collect certain visitor information automatically. It also explains how you can accept or refuse the cookies we serve.
+title: SixSq Websites Cookie Notice
```

Comment:
this doesn't mean we don't need one for nuvla.io right?

Reply:
Good point. Either we add Nuvla.io here or create an entry in the EULA for cookies. It would be easier to do it here.

Reply:
but what I mean is that Nuvla.io needs a cookie notice, that this text does not remove the need for it

Reply:
Indeed. So we need to add it to the list of terms the user must accept when registering.

Reply:
or simply use on the available open-source GDPR cookie notice plugins